clients/geth/specular/rollup/services/sequencer/sequencer.go

```diff
@@ -399,6 +407,22 @@ func (s *Sequencer) confirmationLoop(ctx context.Context) {
 						pendingConfirmationSent = true
 					}
 				}
+			case <-rejectedCh:
+				if !pendingConfirmed {
+					// Reject the first unresolved assertion if all the conditions are met
+					// (1) challenge period has passed, 
+					// (2) at least one staker exists, 
+					// (3) no staker remains staked on the assertion (all have been destroyed).
+					
+					_, err := s.L1Client.RejectFirstUnresolvedAssertion(s.Config.SequencerAddr)
```

Comment:
this doesn't make sense; you're calling reject when you get a reject event?

Reply:
Modifications have been implemented according to feedback, however, rejection criteria is yet to be determined.